src/Library/demos/Column View/main.js

```diff
@@ -0,0 +1,101 @@
+import Gtk from "gi://Gtk";
+import Gio from "gi://Gio";
+
+const column_view = workbench.builder.get_object("column_view");
+
+function createFirstCol() {
+  const factory = new Gtk.SignalListItemFactory();
+  const columnViewColumn = new Gtk.ColumnViewColumn({
+    factory,
+  });
+  factory.connect("setup", (factory, listItem) => {
+    const label = new Gtk.Label({
+      height_request: 50,
```

Comment:
```suggestion
```

Best to keep these at the height of the label, considering how many items are in the list.